src/components/Link/Link.stories.tsx

```diff
@@ -347,3 +347,24 @@ export const IconError: StoryObj<Args> = {
     variant: 'icon',
   },
 };
+
+export const LinkInParagraphContext = {
+  render: () => (
+    <div>
```

Comment:
This needs to be in a `div` because jest doesn't capture the whole story content if it's in a fragment.

Reply:
What's it capture?

Reply:
Before adding the `div`, it only picked up the first line of text before the first `Link` (`Lorem ipsum dolor sit amet,{' '}`).

This is a sidebar but kinda relevant – I also saw a PR on Along where jest wasn't picking up a big section of a component's children, and based on this PR I guessed that it was because a wrapper component was using a fragment, and changing it to a `div` fixed the issue. https://github.com/chanzuckerberg/along/pull/4924#discussion_r942493700

Reply:
😲 this is one of the weirdest things I've ever seen. Feels like Jest/React/TheUniverse shouldn't have any problems with it 🤔